app/assets/javascripts/sequencescape/sequencescape.js

```diff
@@ -34,9 +34,9 @@ window.deselect_all = function(){
   $(scope).find('input[type="checkbox"]:enabled').prop('checked', false);
 }
 
-window.disable_cr_and_change_focus = function(event, current_field, next_field) {
+window.disable_cr_and_change_focus = function(event, next_field) {
   if (event.keyCode !=13 && event.keyCode !=10) { return true; }
```

Comment:
current_field is not used in the actual function. Removed it from here and all places where the function is called.